mms/service.py

```diff
@@ -11,15 +11,16 @@
 """
 CustomService class definitions
 """
-import ast
-from collections import OrderedDict
+import logging
+import time
 
 import mms
 from mms.context import Context
-from mms.log import log_msg
 from mms.metrics.metrics_store import MetricsStore
-from mms.mxnet_model_service_error import MMSError
-from mms.utils.validators.validate_messages import ModelWorkerMessageValidators
+from mms.protocol.otf_message_handler import create_predict_response
+
+INFERENCE_METRIC = 'InferenceTime'
```

Comment:
RunTime will be a more accurate name here? Since we do not know the seperation between preprocess and actual inference?

Reply:
How about PredictionTime

Reply:
That sounds good